lib/trino-parquet/src/main/java/io/trino/parquet/crypto/ModuleType.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.parquet.crypto;
+
+public enum ModuleType
+{
+    Footer((byte) 0),
+    ColumnMetaData((byte) 1),
+    DataPage((byte) 2),
+    DictionaryPage((byte) 3),
+    DataPageHeader((byte) 4),
+    DictionaryPageHeader((byte) 5),
+    ColumnIndex((byte) 6),
+    OffsetIndex((byte) 7),
```

Comment:
Are the column and offset index cases tested ?
Trino writer doesn't write them, so you'll have to rely on ExampleParquetWriter or resource files for that